README.md

```diff
@@ -4,14 +4,13 @@ MUSIC BLOCKS
 Music Blocks is a collection of manipulative tools for exploring
 fundamental musical concepts in an integrative and fun way.
 
-<img
```

Comment:
I have never seen the bug you are seeing. I appreciate that there is a different way to show images in Markdown and maybe this is preferred, but I am curious about the bug itself, since we use <img> for images throughout the documentation.